addon/components/au-modal.js

```diff
@@ -16,6 +21,8 @@ export default class AuModal extends Component {
       'au-modal: No target element was found. Please add the `<AuModalContainer />` component where appropriate.',
       this.destinationElement
     );
+
+    this.closeTriggers = this.args.closeTriggers ? {...CLOSE_TRIGGERS, ...this.args.closeTriggers} : CLOSE_TRIGGERS;
```

Comment:
I've used `closeTriggers` as a variable name here, but I'm not entirely sure this is the best possible option. Something like `canClose` could also suffice?